libcudacxx/include/cuda/__memory_resource/resource.h

```diff
@@ -44,11 +42,21 @@ _LIBCUDACXX_CONCEPT_FRAGMENT(
     requires(_CUDA_VSTD::same_as<void, decltype(__res.deallocate(__ptr, __bytes, __alignment))>),
     requires(_CUDA_VSTD::equality_comparable<_Resource>)));
 
+//! @brief The \c resource concept verifies that a type Resource satisfies the basic requirements of a memory
+//! resource
+//! @rst
+//! We require that a resource supports the following interface
+//!
+//!   - ``allocate(size_t bytes, size_t alginment)``
+//!   - ``deallocate(void* ptr, size_t bytes, size_t alginment)``
+//!   - ``T() == T()``
+//!   - ``T() != T()``
+//!
+//! @endrst
+//! @tparam Resource The type that should implement the resource concept
 template <class _Resource>
```

Comment:
```suggestion
//! @tparam _Resource The type that should implement the resource concept
template <class _Resource>
```

Reply:
I try to use non-ugly names when possible

Reply:
I understand, but doesn't Doxygen or clang-tidy give warnings about it? Clion flags such parameter documentations because there are no parameters named `Resource` in the template parameter list.